packages/social-connection/schemas/SocialConnection.json

```diff
@@ -42,5 +47,9 @@
         "$ref": "#/$defs/connection"
       }
     }
-  }
+  },
+  "additionalProperties": false,
```

Comment:
Could you add this to the `Connection` object as well please to avoid unexpected fields getting added?

Reply:
done